adrs/php/0002_module_structure.md

```diff
@@ -0,0 +1,30 @@
+# Module structure
+
+## Motivation
+
+This ADR standardizes the design of different modules.
+A module divides this project into smaller parts.
+Each module defines a set of classes that belong together and are independently of classes in other modules.
+
+## Decision
+
+### Follow Package design
```

Comment:
I'm not very happy with this section:
1. What should the headline "Follow Package design" mean? Isn't "Package design" enough
2. The section starts with a list out of nowhere. There is not introduction or anything else. When I read it I have no clue what you want to say with it.
3. The rest of the text is very opinionated. "We want to ...", "We stand for ..." sounds more like a marketing text than a qualified decision log.

Do you think you can improve it?

Reply:
Yes, for sure. It was a brief idea looking for some feedback, thanks! I will rethink about it 👍🏼

Reply:
This ADR is fully related to this PR https://github.com/phel-lang/phel-lang/pull/253 
Once that PR is resolved, we'll be able to finish this ADR text :)

Reply:
I will remove this `0002_module_structure.md` for now, and leave in this PR only the other one `0003_compiler_module.md`. This way, I'll have some more time to think about the other one and see if I can get something better :)